docs/OV_Runtime_UG/migration_ov_2_0/intro.md

```diff
@@ -77,5 +78,6 @@ Please look at next transition guides to understand how migrate Inference Engine
  - [Preprocess your model](./preprocessing.md)
  - [Configure device](./configure_devices.md)
  - [OpenVINO™ Model Creation](graph_construction.md)
+ - [Install & deployment](deployment_migration.md)
```

Comment:
I'm wondering if we should set this as first doc in list? Given that logically this is the first thing you do (aligned with Get Started)